app/src/androidTest/java/com/github/sdp_begreen/begreen/firebase/FirebaseDBTest.kt

```diff
@@ -159,4 +167,42 @@ class FirebaseDBTest {
         }
     }
 
+    @Test
+    fun storeBinLocationReturnsTrue() {
+        runBlocking {
+            assertTrue(FirebaseDB.storeBinLocation(LatLng(1.1, -2.2)))
+        }
+    }
+
+    @Test
+    fun storeBinLocationCorrectlyUpdatesDatabase() {
+
+        val binLocation = LatLng(12.3, -43.0)
+
+        runBlocking {
+
+            assertTrue(FirebaseDB.storeBinLocation(binLocation))
```

Comment:
No it's fine. I said that only because a team member suggested me to do like that but if the project already contains both of them there is not problem